### ISSUE TITLE
TypeError shows up on console during e2e test execution or before Lightspeed login

### ISSUE THREAD

**Issue by TamiTakamiya** [ansible/vscode-ansible#1063]
### Summary

TypeError shows up on console during e2e test execution. 

Note: This was found while working on #1062.  It was fixed as [a part of the PR](https://github.com/ansible/vscode-ansible/pull/1062/commits/70182c9a0e4a682488be7131f3573bc419b7b856), but i decided to handle as a separate issue [per review comment](https://github.com/ansible/vscode-ansible/pull/1062/commits/70182c9a0e4a682488be7131f3573bc419b7b856#r1465599842).

Following is the error shows up on console output.

```
rejected promise not handled within 1 second: TypeError: Cannot read properties of undefined (reading 'rhUserHasSeat')
stack trace: TypeError: Cannot read properties of undefined (reading 'rhUserHasSeat')
        at getLoggedInSessionDetails (/home/ttakamiy/git/ansible/vscode-ansible/out/client/src/features/lightspeed/utils/webUtils.js:1220:19)
        at LightspeedStatusBar.setLightSpeedStatusBarTooltip (/home/ttakamiy/git/ansible/vscode-ansible/out/client/src/features/lightspeed/statusBar.js:2766:95)
```

### Extension version

2.10.130

### VS Code version

1.85.1

### Ansible Version

```shell
ansible [core 2.16.1]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/ttakamiy/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/ttakamiy/.local/lib/python3.12/site-packages/ansible
  ansible collection location = /home/ttakamiy/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/ttakamiy/.local/bin/ansible
  python version = 3.12.0 (main, Oct  2 2023, 00:00:00) [GCC 13.2.1 20230918 (Red Hat 13.2.1-3)] (/usr/bin/python3)
  jinja version = 3.1.2
  libyaml = True
```

### OS / Environment

Fedora 39

**Comment by TamiTakamiya** [comment]
@ssbarnea I have not fixed this in the linked PRs and I am still seeing the issues when lightspeed e2e tests are being executed. I am not sure why you do not see in your enviornment. The root cause is that we are using a predefined auth token in our e2e test and skip the login flow. I think we can mock that flow and it's what I want to implement with this issue. It is also needed for supporting some lightspeed use cases in e2e tests (e.g. to ensure some features are enabled only for seated users).

**Comment by TamiTakamiya** [comment]
While working on another issue, I realized that those TypeErrors show up in non-test environment as well when user has not logged in to Lightspeed server (hence there is no session yet). I have updated the title of this issue accordingly.

**Comment by TamiTakamiya** [comment]
This issue was addressed by a change included in #1088.  I am closing this.
